local-volume/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,24 @@
+common:
+  namespace: default
+  defaultMountDir: /local-disks
+configmap:
+  configMapName: "test"
+  storageClass:
+    - local-fast:
+        hostDir: "/mnt/fast"
+        mountDir: "/local-fast"
+    - local-normal:
+        hostDir: "/mnt/normal"
+        mountDir: "/local-normal"
+    - local-slow:
+        hostDir: "/mnt/slow"
+daemonset:
+  name: "local-volume-provisioner"
+  image: quay.io/external_storage/local-volume-provisioner:latest
+  imagePullPolicy: Always
+  serviceAccount: local-storage-admin
+  kubeConfigEnv: 
```

Comment:
Can these actually be removed (commented out), or is it required to be empty?

Reply:
done